src/SFA.DAS.FAT.Web.AcceptanceTests/Features/CourseDetail.feature

```diff
@@ -2,28 +2,30 @@
 	As an employer
 	I want to know if the start date has passed for a bookmarked course
 	So that I can know it is no longer available
-
+	
+@WireMockServer
 Scenario: Course detail page
-When I navigate to the following url: /courses/14
-Then an http status code of 200 is returned
-And the page content includes the following: Apprenticeship training course
-#todo: and course details are displayed
-And the expired course content is not displayed
-And the last start date alert is not displayed
+	Given I navigate to the following url: /courses/14
+	Then an http status code of 200 is returned
+	And the page content includes the following: Apprenticeship training course
+	And the expired course content is not displayed
+	And the last start date alert is not displayed
 
+@WireMockServer
 Scenario: Course with last start date
-When I navigate to the following url: /courses/24 
-Then an http status code of 200 is returned
-And the last start date alert is displayed
-
+	Given I navigate to the following url: /courses/24 
+	Then an http status code of 200 is returned
+	And the last start date alert is displayed
+	
+@WireMockServer	
 Scenario: Expired course
-When I navigate to the following url: /courses/101
-Then an http status code of 200 is returned
-And the expired course content is displayed
-And the last start date alert is not displayed
-#todo: and course details are not displayed
+	Given I navigate to the following url: /courses/101
+	Then an http status code of 200 is returned
+	And the expired course content is displayed
+	And the last start date alert is not displayed
 
+@WireMockServer
 Scenario: Regulated course
-When I navigate to the following url: /courses/333
-Then an http status code of 200 is returned
-And the regulated occupation header and message is displayed
\ No newline at end of file
+	Given I navigate to the following url: /courses/333
+	Then an http status code of 200 is returned
+	And the regulated occupation header and message is displayed
```

Comment:
If we're checking that the inappropriate content isn't shown in the first test, should we not check that in the other 2 tests here?

Reply:
I've refactored this so its only checking the one thing that its actually interested in 😄